src/py/flwr/cli/new/new.py

```diff
@@ -0,0 +1,125 @@
+# Copyright 2024 Flower Labs GmbH. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Flower command line interface `new` command."""
+
+import os
+from enum import Enum
+from string import Template
+from typing import Dict, Optional
+
+import typer
+from typing_extensions import Annotated
+
+from ..utils import prompt_options
+
+
+class MlFramework(str, Enum):
+    """Available frameworks."""
+
+    PYTORCH = "PyTorch"
+    TENSORFLOW = "TensorFlow"
+
+
+class TemplateNotFound(Exception):
+    """Raised when template does not exist."""
+
+
+def load_template(name: str) -> str:
+    """Load template from template directory and return as text."""
+    tpl_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), "templates"))
+    tpl_files = os.listdir(tpl_dir)
```

Comment:
Are template files going to get packaged by our build process (i.e., does this continue to work when installing a whl)?

Reply:
@danieljanes yes. You can just do 

```bash
pip install https://s3.eu-central-1.amazonaws.com/artifact.flower.ai/py/pr/flower_cli/173afac/flwr-1.8.0-py3-none-any.whl
```

and test it.